i18n/languages/en.json

```diff
@@ -228,6 +232,20 @@
     "Controls.ToggleScreens.FailedToggle": "Failed to toggle interior screens",
     "Controls.ToggleScreens.Status": "Interior screens",
     "Controls.ToggleScreens.Tip": "Toggle Screen",
+    "Controls.Tracking": "Tracking",
+    "Controls.Tracking.DistanceFail": "Can't track entities that are too far away",
+    "Controls.Tracking.FlightFail": "Failed to to enable flight mode for tracking",
+    "Controls.Tracking.GenericFail": "Failed to track entity",
+    "Controls.Tracking.InteriorFail": "Can't track entities inside the TARDIS",
+    "Controls.Tracking.PilotWarning": "WARNING: Tracking is enabled, press '%s' to change target or disable tracking",
+    "Controls.Tracking.Rotation": "Tracking rotation",
+    "Controls.Tracking.SameTarget": "You are already tracking this entity",
+    "Controls.Tracking.SelfFail": "Can't track itself",
```

Comment:
I would extend it just a bit, probably "The TARDIS can't track itself"

Reply:
https://github.com/MattJeanes/TARDIS/pull/969/commits/2c72beec1d2ad53220acf32989f31d755e0c3e72